telluric/georaster.py

```diff
@@ -2017,8 +2017,8 @@ def chunks(self, shape=256, pad=False):
                 cur_raster = _self.get_window(window)
                 yield RasterChunk(raster=cur_raster, offsets=(col_off, row_off))
 
-    def to_assets(self):
-        return {"0": {"href": self._filename, "bands": self.band_names}}
+    def to_assets(self, **attr):
+        return {"0": dict(href=self._filename, bands=self.band_names, __object=self, type=RASTER_TYPE, **attr)}
```

Comment:
I think the key: `"0"` should aslo be argumented so how, meaning the use should have control on that

and also we need to be sure that:
```
assert dict == GeoRaster.from_assets(dict).to_assets() 
```